ffi/src/lib.rs

```diff
@@ -161,20 +159,15 @@ impl<'itm, T: ReprC> TryFromReprC<'itm> for &'itm mut T {
     }
 }
 
-impl<T: ReprC + Copy> IntoFfi for &T
-where
-    T: IntoFfi<Target = T>,
-{
+impl<T: ReprC + Copy + IntoFfi<Target = T>> IntoFfi for &T {
```

Comment:
ok, let's say defining bounds in one place is more readable. Should we strive to place bounds in the `where` clause or type declaration or as the case requires? I've seen both approaches in this PR. Not that you have to change this now

Reply:
In this case, visually. Here having them all in one place is still under 80 columns.

Reply:
Speaking of which, we should consider this for our contributing guide and style guide